avocado/plugins/runner_nrunner.py

```diff
@@ -61,7 +61,10 @@ def run_suite(self, job, result, test_suite, variants, timeout=0,
                     break
 
             # test execution time is currently missing
-            test_state = {'status': statuses[-1]['status'].upper()}
+            # since 358e800e81 all runners all produce the result in a key called
+            # 'result', instead of 'status'.  But the Avocado result plugins rely
+            # on the current runner approach
+            test_state = {'status': statuses[-1]['result'].upper()}
```

Comment:
Hi @clebergnu, just one question: How difficult it is to fix that on the result plugin? IIUC, this ambiguity should be fixed on the upper layers too, right? Or shouldn't it?

Reply:
The ambiguity is really caused by two different "standards". The current runner uses `status`, while the nrunner recently changed to use `result`. In theory we could change the current runner too, but I don't think that's the right approach at this point.